packages/core/src/components/BubbleList/index.vue

```
@@ -51,91 +59,80 @@ const lastScrollTop = ref(0);
+
 // 组件内部触发方法，跟随打字器滚动，滚动底部
 function autoScroll() {
-  if (scrollContainer.value) {
-    const listBubbles = scrollContainer.value!.querySelectorAll(
-      '.el-bubble-content-wrapper'
-    );
-    // 如果页面上有监听节点，先移除
-    if (resizeObserver.value) {
-      resizeObserver.value.disconnect();
+  const container = scrollContainer.value;
+  if (!container) return;
```

ESLint

[error] Expect newline after if

(antfu/if-newline)